ncm-spma/src/main/pan/components/spma/schema.pan

```diff
@@ -40,13 +40,15 @@ type SOFTWARE_REPOSITORY_PROTOCOL = {
 type SOFTWARE_REPOSITORY = {
     "enabled" : boolean = true
     "gpgcheck" : boolean = false
+    "mirrorlist" : boolean = false
     "excludepkgs" ? string[]
     "includepkgs" ? string[]
-    "name" ? string with match(SELF, '^[\w-.]+$') # "Repository name"
+    "name" ? string  # "Repository name"
```

Comment:
Actually I will resurrect the limitation as it makes sense - yum doesn't like '[' or ']' in repo name.